data_test.go

```diff
@@ -91,6 +92,16 @@ func Test_Data(t *testing.T) {
 				t.Error(err)
 			}
 			t.Run("virtual machines", func(t *testing.T) {
+				t.Run("match capability logic", func(t *testing.T) {
+					sku, err := cache.Get(ctx, "standard_d4s_v3", VirtualMachines, "eastus")
+					if err != nil {
+						t.Errorf("expected to find virtual machine sku standard_d4s_v3")
+					}
+
+					match := Match(ctx, cache, &sku, "eastus")
```

Comment:
this currently returns zero matches because == is too strict on capabilities, but you get the idea..